templates/repo/issue/new_form.tmpl

```diff
@@ -34,52 +34,7 @@
 	<div class="four wide column">
 		<div class="ui segment metas">
 			{{template "repo/issue/branch_selector_field" .}}
-
-			<input id="label_ids" name="label_ids" type="hidden" value="{{.label_ids}}">
```

Comment:
`<input id="label_ids" name="label_ids"` is lost in the common template.

Newly created issues won't have labels?

Reply:
Thanks a lot for catching that.